addons/hr_timesheet/migrations/10.0.1.0/pre-migration.py

```diff
@@ -29,7 +29,32 @@ def create_and_populate_department(cr):
     ''')
 
 
+def create_and_populate_children(cr):
+    openupgrade.logged_query(cr, '''
+    ALTER TABLE project_task ADD COLUMN parent_id INT;
+
+    -- Since the parent_id is new there's no point in computing the
```

Comment:
Why this explanation is part of the logged query?

Reply:
I could make it outside the query, but I think the comment belongs near to the "funny" code.   I like to leave my thoughts as clear as I deem possible.  In this case, the reason behind why I can put `children_hours=0` is not quite obvious.